Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.H

```diff
@@ -29,8 +29,12 @@ class SpectralFieldDataRZ
         using FFTplans = amrex::LayoutData<cufftHandle>;
```

Comment:
This could be cleaned up by using `AnyFFT`,
```
using FFTplans = amrex::LayoutData<AnyFFT::VendorFFTPlan>
```

Reply:
Good idea, done :)